timeseries/tests/smoketests/test_all_models.py

```diff
@@ -182,8 +181,43 @@ def test_all_models_can_handle_all_covariates(
     assert predictions.index.equals(future_test_data.index)
 
 
-@pytest.mark.parametrize("freq", DEFAULT_SEASONALITIES.keys())
-def test_all_models_handle_all_pandas_frequencies(freq, all_model_hyperparams):
+@pytest.mark.parametrize(
+    "freq",
+    [
+        "YE",
+        "QE",
+        "SME",
+        "W",
+        "D",
+        "B",
+        "bh",
+        "h",
+        "min",
+        "s",
+    ],
+)
+@pytest.mark.parametrize(
+    "hyperparameters",
+    [
+        {"Chronos": {"model_path": "autogluon/chronos-bolt-350k-test"}},
+        {"DLinear": DUMMY_MODEL_HPARAMS},
+        {"DeepAR": DUMMY_MODEL_HPARAMS},
+        {"DirectTabular": DUMMY_MODEL_HPARAMS},
+        {"AutoETS": {**DUMMY_MODEL_HPARAMS, "model": "AAA"}},
+        {"NPTS": DUMMY_MODEL_HPARAMS},
+        {"Naive": DUMMY_MODEL_HPARAMS},
+        {"PatchTST": DUMMY_MODEL_HPARAMS},
+        {"RecursiveTabular": DUMMY_MODEL_HPARAMS},
+        {"SeasonalAverage": DUMMY_MODEL_HPARAMS},
+        {"SeasonalNaive": DUMMY_MODEL_HPARAMS},
+        {"SimpleFeedForward": DUMMY_MODEL_HPARAMS},
```

Comment:
I think we can drop SimpleFeedForward and WaveNet here as well since these model are not used in any presets.

Reply:
again, will leave them in as they do not hurt greatly in runtime.